test/system/800-scenario.bats

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bats 
+#
+# Test for the scenario that
+# "podman import" tarball exported from a container and modified the content.
+# 
+
+load helpers
+
+alpine="quay.io/libpod/alpine"
```

Comment:
There's already an internal variable `$IMAGE` which other tests are using.  I suggest using that instead of `$alpine` to reduce the number of pulls and hence speed up the test a bit.